s3_helper.py

```diff
@@ -0,0 +1,70 @@
+import boto3
+import botocore.exceptions
+import json
+import logging
+import traceback
+
+
+class S3Helper(object):
+
+    def __init__(self, aws_profile=None, logger=False):
+        self.aws_profile = aws_profile
+        self.print_func = print
+        if logger:
+            self.print_func = logger.info
+
+        self.client = self._get_client()
+
+    def _create_aws_session(self):
+        try:
+            if self.aws_profile:
+                session = boto3.session.Session(profile_name=self.aws_profile)
+            else:
+                session = boto3.session.Session()
+        except botocore.exceptions.ProfileNotFound:
+            self.print_func('Please supply a valid AWS profile name.')
+            exit()
+        except:
+            self.print_func(traceback.format_exc())
```

Comment:
Should we exit here if another exception occurs?

Reply:
Good point. Adding additional error message and exiting here.